src/esp/assets/Attributes.cpp

```diff
@@ -40,10 +40,12 @@ PhysicsObjectAttributes::PhysicsObjectAttributes(
   setInertia({0, 0, 0});
   setLinearDamping(0.2);
   setAngularDamping(0.2);
-  // default collisions will be mesh for physics objects
+  // default rendering and collisions will be mesh for physics objects
   // primitive-based objects do not currently support mesh collisions, however,
   // due to issues with how non-triangle meshes (i.e. wireframes) are handled in
   // @ref GenericMeshData::setMeshData
+  setRenderAssetIsPrimitive(false);
+  setCollisionAssetIsPrimitive(false);
```

Comment:
As this is currently configured, these flags should only be set contextually based on asset handles set by the user, and therefore should not be exposed to bindings (since user set values would be ignored anyway).